iron-validatable-behavior.html

```diff
@@ -114,12 +94,19 @@
      * your element to have custom validation logic, do not override this method;
      * override `_getValidity(value)` instead.
 
-     * @param {Object} value The value to be validated. By default, it is passed
-     * to the validator's `validate()` function, if a validator is set.
+     * @param {Object} value Deprecated: The value to be validated. By default,
+     * it is passed to the validator's `validate()` function, if a validator is set.
+     * If this argument is not specified, then the element's `value` property
+     * is used, if it exists.
      * @return {boolean} True if `value` is valid.
      */
     validate: function(value) {
-      this.invalid = !this._getValidity(value);
+      // If this is an element that also has a value property, and there was
+      // no explicit value argument passed, use the element's property instead.
+      if (value === undefined && this.value !== undefined)
+        this.invalid = !this._getValidity(this.value);
+      else
+        this.invalid = !this._getValidity(value);
```

Comment:
Oh, maybe `value` is `null` sometimes and that needs to be passed to `_getValidity`?